locales/en.json

```diff
@@ -8,13 +8,49 @@
     "audiobooks": "Audiobooks",
     "search": "Search"
   },
+  "sidebar": {
+    "playlists": {
+      "title": "Playlists"
+    }
+  },
+  "profile": {
+    "title": "Profile",
+    "autoplay": { "title": "Auto Play" },
```

Comment:
I'm not sure if I like the title part. Why not call it autoplay directly?

Reply:
Unless you give a good reason, we should change it.
It's been done differently everywhere else

Reply:
I did it because I expect a description to be added, as we have it on the app. To mark to the translator that this is a title, having a more detailed description, I did it this way.